pyiron_core/pyironflow/treeview.py

```diff
@@ -67,6 +67,16 @@ def __init__(self, root_path=None, flow_widget=None, log=None, layout=None):
             import pyiron_core.pyiron_nodes as pyiron_nodes
 
             root_path = pyiron_nodes.__path__[0]
+            self.node_path = "pyiron_core"
```

Comment:
This seems to be inconsistent with the logic in the other branches or am I missing something?
`pyiron_nodes.__path__[0]` would evaluate e.g. to `/home/ponder/science/phd/dev/pyiron_core/pyiron_core/pyiron_nodes` and in the other branches we take the last part of that part, so `pyiron_nodes` rather than `pyiron_core`.  Shouldn't it be this then?

```suggestion
            self.node_path = "pyiron_nodes"
```

Reply:
While I agree that it would be more consistent to use `pyiron_nodes` this would break backwards compatibility to all previously saved workflows. The current changes introduce new functionality - the option to specify a user defined path - without breaking the backwards compatibility. If no path is specified the code falls back to the previous behavior.